packages/clay-pagination-bar/src/__tests__/index.tsx

```diff
@@ -78,4 +78,31 @@ describe('ClayPaginationBar', () => {
 			document.body.querySelector('.dropdown-menu')!.classList
 		).toContain('show');
 	});
+
+	it('automatically go to page 1 if active page exceeds delta', () => {
+		const Comp = () => {
+			const [activePage, setActivePage] = React.useState(2);
+			const [delta, setDelta] = React.useState(5);
+
+			return (
+				<ClayPaginationBarWithBasicItems
+					activeDelta={delta}
+					activePage={activePage}
+					onDeltaChange={setDelta}
+					onPageChange={setActivePage}
+					spritemap={spritemap}
+					totalItems={15}
+				/>
+			);
+		};
+		const {container, getByTestId} = render(<Comp />);
+
+		fireEvent.click(getByTestId('selectPaginationBar'), {});
+
+		fireEvent.click(getByText(document.body, '20 items'), {});
+
+		expect(getByText(container, '1').parentElement?.classList).toContain(
```

Comment:
Hmm, that is odd. I was going off of ts complaints which is why I added it. I'll see if I can remove it

Reply:
Types:

`getByText()` returns `HTMLElement ` (ts doesn't account for the throw)

`HTMLElement.parentElement` is `HTMLElement | null`

So it makes sense why TS would complain. Do you think its worth trying to avoid the `?.`?

Reply:
> Do you think its worth trying to avoid the `?.`

Nope. My bad. Too much time in liferay-portal that I forgot this was TS.

Even though *we* know it's never going to try accessing undefined, TS doesn't, so it's fair.